docs/multicluster/architecture.md

```diff
@@ -106,3 +108,52 @@ Multi-cluster admins can also specify certain ClusterNetworkPolicies to be repli
 ClusterSet. The ACNP to be replicated should be created as a ResourceExport in the leader cluster, and
 the resource export/import pipeline will ensure member clusters receive this ACNP spec to be replicated.
 Each member cluster's Multi-cluster Controller will then create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Service Connectivity
+
+Antrea Multi-cluster supports Service connectivity directly since 1.7.0. It introduces a new feature gate
+in Antrea Agent which is called `Multicluster`. When Antrea Agent is deployed with `Multicluster`
+feature, user can choose one of K8s Nodes as the Gateway in a member cluster. The assigned Gateway Node
+will be responsible to forward all cross-clusters traffic from local cluster to other member clusters.
+Below is the basic architecture for Antrea Multi-cluster Service network connectivity.
+
+<img src="assets/mc-service-connectivity.svg" width="600" alt="Antrea Multi-cluster Service Connectivity">
```

Comment:
Again, a little small.

Reply:
We should remove "node-a1" around the tunnel line?

A minor one - the color of Cluster C box looks too different from that of Cluster A B. Could we adjust the colors?

Reply:
yea, node-a1 is added by mistake, it's fixed, and color is adjusted.